src/main/java/com/mindee/parsing/proofofaddress/ProofOfAddressV1DocumentPrediction.java

```diff
@@ -0,0 +1,93 @@
+package com.mindee.parsing.proofofaddress;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.mindee.parsing.SummaryHelper;
+import com.mindee.parsing.common.field.CompanyRegistrationField;
+import com.mindee.parsing.common.field.DateField;
+import com.mindee.parsing.common.field.LocaleField;
+import com.mindee.parsing.common.field.StringField;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Getter
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProofOfAddressV1DocumentPrediction {
+
+  /**
+   * Locale information.
+   */
+  @JsonProperty("locale")
+  private LocaleField locale;
+  /**
+   * ISO date yyyy-mm-dd. Works both for European and US dates.
+   */
+  @JsonProperty("date")
+  private DateField issuanceDate;
+  /**
+   * All extracted ISO date yyyy-mm-dd. Works both for European and US dates.
+   */
+  @JsonProperty("dates")
+  private List<DateField> dates = new ArrayList<>();
+  /**
+   * Name of the document's issuer.
+   */
+  @JsonProperty("issuer_name")
+  private StringField issuerName;
+  /**
+   * Address of the document's issuer.
+   */
+  @JsonProperty("issuer_address")
+  private StringField issuerAddress;
+  /**
+   * Generic: VAT NUMBER, TAX ID, COMPANY REGISTRATION NUMBER or country specific.
+   */
+  @JsonProperty("issuer_company_registrations")
+  private List<StringField> issuerCompanyRegistrations = new ArrayList<>();
+  /**
+   * Name of the document's recipient.
+   */
+  @JsonProperty("recipient_name")
+  private StringField recipientName;
+  /**
+   * The surname of the person.
+   */
+  @JsonProperty("recipient_address")
+  private StringField recipientAddress;
+  /**
+   * Generic: VAT NUMBER, TAX ID, COMPANY REGISTRATION NUMBER or country specific.
+   */
+  @JsonProperty("recipient_company_registrations")
+  private List<CompanyRegistrationField> recipientCompanyRegistrations = new ArrayList<>();
+
+  @Override
+  public String toString() {
+
+    String summary =
+        String.format(":Locale: %s%n", this.getLocale()) +
+        String.format(":Issuer name: %s%n", this.getIssuerName()) +
+        String.format(":Issuer Address: %s%n", this.getIssuerAddress()) +
+        String.format(":Issuer Company Registrations: %s%n",
+          this.issuerCompanyRegistrations.stream()
+            .map(StringField::toString)
+            .collect(Collectors.joining(" "))) +
```

Comment:
Helper function for this? would cut down on duplicated code

Reply:
It will be useful if I had an object to encapsulate the list of company registrations.
Of course we could create even so, but if I create a helper to this one, I should do the same for all the other stream used.

Reply:
yes, the idea would be to call a generic function on all list types that just concatenates their `toString` output.

Maybe be able to pass the separator, like `" "` or `"\n"`...

Not at all required for this PR, just something to think about.

Reply:
Okay got it, I will work on something